site/src/Tour.jsx

```diff
@@ -35,6 +36,24 @@ const Steps = [
     placement: "auto",
     offset: 0,
   },
+  {
+    target: ".tour-layers-checkboxes",
+    content:
+      "Altering the visibility at the theme level is available, as well as at a more granular type level",
```

Comment:
I prefer "Visibility Toggles are available at both the 'theme' and 'type' levels"